sceptre/resolvers/stack_output.py

```diff
@@ -163,21 +173,43 @@ def resolve(self):
         """
         self.logger.debug("Resolving external Stack output: {0}".format(self.argument))
 
-        profile = None
-        region = None
-        sceptre_role = None
         arguments = shlex.split(self.argument)
 
+        if not arguments:
+            message = "!stack_output_external requires at least one argument"
+            raise SceptreException(message)
+
         stack_argument = arguments[0]
```

Comment:
It doesn't look like you're still using `stack_argument`

Reply:
Yes, it's used on the next line.